CONTRIBUTING.md

```diff
@@ -37,6 +37,10 @@ This section should get you running with **Amplify CLI** and get you familiar wi
 
    > If you are using Yarn v2, run `yarn set version classic` to change to Yarn Classic.
 
+   > Ensure that [Yarn global bin](https://classic.yarnpkg.com/en/docs/cli/global) is added to your PATH. For example, add `export PATH="$(yarn global bin):$PATH"` to your shell profile file on Linux or macOS.
+
+1. Ensure you have [Java](https://adoptium.net/) installed and `java` command is available in your system. This is required for DynamoDB emulator.
```

Comment:
Might be better to point people here: https://aws.amazon.com/corretto/ to download java

Reply:
Sorry @sobolk, I should have pointed you here ^